config/crds/cluster_v1alpha1_machine.yaml

```diff
@@ -42,6 +42,22 @@ spec:
                 to indicate what labels, annotations, name prefix, etc., should be
                 used when creating the Node.
               type: object
+            providerID:
```

Comment:
Do these changes get clobbered if we change the apis and re-run make?

Reply:
Yeah that's correct. Ideally we should check this in CI like we do for the clientset.